client/my-sites/stats/stats-navigation/index.jsx

```diff
@@ -13,9 +17,12 @@ import NavItem from 'components/section-nav/item';
 import FollowersCount from 'blocks/followers-count';
 import SegmentedControl from 'components/segmented-control';
 import QueryJetpackPlugins from 'components/data/query-jetpack-plugins';
+import { isPluginActive } from 'state/selectors';
+import { isJetpackSite } from 'state/sites/selectors';
+import { UNITS as StoreStatsTabs } from 'extensions/woocommerce/app/store-stats/constants';
```

Comment:
Thanks @mcsf, I didn't notice that. Looks like something we'll need to clean up.

/cc @psealock